README.md

```diff
@@ -80,5 +80,9 @@ might be running versions different from the vendored one, creating hard to debu
 To run integration tests locally, you should set the `INTEGRATION` environment variable. The dependencies of the integration tests are best run via Docker. To get an
 idea about the versions and the set-up take a look at our [CI config](./.circleci/config.yml).
 
-The best way to run the entire test suite is using the [CircleCI CLI](https://circleci.com/docs/2.0/local-jobs/). Simply run `circleci build`
-in the repository root. Note that you might have to increase the resources dedicated to Docker to around 4GB.
+The best way to run the entire test suite is using the [CircleCI CLI](https://circleci.com/docs/2.0/local-cli/). In order to run
```

Comment:
Side-note: I updated those instructions to help a bit with the local development/testing but I ended up going the route of having my own docker-compose setup with mysql, postgres and sqlserver because I wanted to be able to repeatedly run tests and even run tests in debug mode. I wonder if we could improve the tooling here without adding too much maintenance cost.